src/components/media_manager/src/media_manager_impl.cc

```diff
@@ -341,8 +327,63 @@ const MediaManagerSettings& MediaManagerImpl::settings() const {
 }
 
 uint32_t MediaManagerImpl::DataSizeToMilliseconds(uint64_t data_size) const {
+  uint32_t bits_per_sample = 16;
+  uint32_t sampling_rate = 16000;
+
+  if (application_manager_.hmi_capabilities().pcm_stream_capabilities()) {
+    const auto pcm_caps =
+        application_manager_.hmi_capabilities().pcm_stream_capabilities();
+
+    if (pcm_caps->keyExists(application_manager::strings::bits_per_sample)) {
+      auto type = static_cast<hmi_apis::Common_BitsPerSample::eType>(
+          pcm_caps->getElement(application_manager::strings::bits_per_sample)
+              .asUInt());
+      switch (type) {
+        case hmi_apis::Common_BitsPerSample::RATE_8_BIT: {
+          bits_per_sample = 8;
+          break;
+        }
+        case hmi_apis::Common_BitsPerSample::RATE_16_BIT: {
```

Comment:
Why is this the case if the variable uint32_t bits_per_sample = 16; has this number in the declaration?

Reply:
If someone changes magic number and forgets to add  this case, it will be an issue.